BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -10,146 +10,167 @@ import Foundation
 import UIKit
 import SWRevealController
 
-
-
-
-protocol SideMenuFilterViewControllerDelegate {
-
-    func onButtonTap(sender: UIButton)
-
+protocol SideMenuFilterDelegate {
+    
+    func didApplyFilters(_ filter: SideMenuFilterViewController, restaurants: Bool, stores: Bool, openNow: Bool, distanceInMiles: Int)
+    func didCancelFilters()
+    
 }
 
-
-class SideMenuFilterViewController: UITableViewController, SWRevealViewControllerDelegate {
-
+class SideMenuFilterViewController: UITableViewController {
+    
     @IBOutlet weak var slider: UISlider!
     @IBOutlet weak var distanceLabel: UILabel!
     @IBOutlet weak var openNowLabel: UILabel!
     @IBOutlet weak var openNowSwitch: UISwitch!
     @IBOutlet weak var restaurantButton: UIButton!
     @IBOutlet weak var storesButton: UIButton!
     @IBOutlet weak var applyButton: UIButton!
-    
     @IBOutlet weak var cancelButton: UIButton!
     @IBOutlet weak var contentViewCell: UIView!
-
+    @IBOutlet weak var slideValueLabel: UILabel!
+    
+    var distanceFilter = 0
+    var isRestaurant = false
+    var isStore = false
+    var isOpenNow = false
+    var delegate: SideMenuFilterDelegate?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        setButtonAttributes()
+        
+        styleMenu()
+        
     }
-
-
-
-    public var distanceFilter: Double?
-    var delegate: SideMenuFilterViewControllerDelegate?
-    var hoursOfOperation: Bool?
-    var isRestaurant: Bool?
```

Comment:
These values should not be optional, but should instead come loaded with default values